sdk/identity/azure-identity/pom.xml

```diff
@@ -90,7 +96,7 @@
     <dependency>
       <groupId>net.minidev</groupId>
       <artifactId>json-smart</artifactId>
-      <version>2.4.7</version> <!-- {x-version-update;net.minidev:json-smart;external_dependency} -->
+      <version>2.4.8</version> <!-- {x-version-update;net.minidev:json-smart;external_dependency} -->
```

Comment:
msal4j currently uses version 2.4.7
We should ask msal4j to upgrade to latest oauth2-oidc-sdk first to ensure it is compatible with json-smart:2.4.8 before upgrading the pinned version here.

Reply:
ok, will remain 2.4.7 this pr meanwhile ask msal4j to upgrade oauth2-oidc-sdk